_posts/2024-05-23-geoserver-2-25-1-released.md

```diff
@@ -0,0 +1,121 @@
+---
+author: Jody Garnett
+date: 2024-05-23
+layout: post
+title: GeoServer 2.25.1 Release
+categories:
+- Announcements
+tags:
+- Release
+- Vulnerability
+release: release_225
+version: 2.25.1
+jira_version: 16919
+--- 
+
+GeoServer [2.25.1](/release/2.25.1/) release is now available
+with downloads
+([bin](https://sourceforge.net/projects/geoserver/files/GeoServer/2.25.1/geoserver-2.25.1-bin.zip/download),
+[war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.25.1/geoserver-2.25.1-war.zip/download),
```

Comment:
I do not believe that Sourceforge latests have been updated to 2.25.1? https://sourceforge.net/projects/geoserver/files/latest/download

Reply:
not sure how latest works?

Reply:
Yes, but https://sourceforge.net/projects/geoserver/files/latest/download downloads geoserver-2.25.0-war.zip

Fixed now, as per

![image](https://github.com/geoserver/geoserver.github.io/assets/5699667/82e1bfa3-a1b4-4e39-96bb-7719d197f81d)